changelog/woopmnt-5282-implement-all-elements-with-data-from-the-server

```diff
@@ -0,0 +1,4 @@
+Significance: minor
+Type: add
+
+New Account Details: implement complete component
```

Comment:
Maybe I'm getting petty, but can we make this a bit flasier? Merchants/devs would not be aware of the currently available unfinished component.
```suggestion
Redesigned Account Status component for better clarity and more flexibility 
```